meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-startWorld.sh

```
@@ -60,12 +213,47 @@ echo "Running image/container...."  > /dev/tty3
+    READONLY_OPTS="--read-only --tmpfs /etc/ssl/certs:rw,nosuid,nodev,noexec,size=8m -v /docker_persist/.globalping-probe-uuid:/.globalping-probe-uuid:rw"
+    echo "Container will run in read-only mode" > /dev/tty3
+else
+    READONLY_OPTS=""
+    echo "Container will run in read-write mode" > /dev/tty3
+fi
+
+# First container start attempt (before main loop)
+# This ensures container is running before systemMonitor starts checking
+echo "Starting container for the first time..." > /dev/tty3
+docker rm -f globalping-probe 2>/dev/null
+/usr/bin/docker run -d $TMPFS_OPTS $READONLY_OPTS --env GP_HOST_HW --env GP_HOST_DEVICE --env GP_HOST_FIRMWARE --env GP_PROBE_UUID --log-driver local --log-opt max-size=10m --network host --restart=always --name globalping-probe globalping/globalping-probe
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)